CONTRIBUTING.md

```diff
@@ -1,6 +1,32 @@
-# Web World Wind Design and Coding Guidelines
+# Contributing to Web WorldWind
 
-## General
+The following are a set of guidelines for contributing to the NASA Web WorldWind project. These are mostly guidelines, not
```

Comment:
I have mixed opinions about inviting changes to our standards. The design and coding guidelines, for example, are principles we adhere to. Would you please give me some insight into your primary audience for these opening sentences, and the motivation behind them?